dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -439,6 +439,16 @@ public void testBomCoordinatesInUnstableArtifacts() throws IOException, ParsingE
         "BOM: com.google.cloud:libraries-bom:1.0.0", bomCoordinatesNodes.get(0).getValue());
   }
 
+  @Test
+  public void testDependencyTrees() throws IOException, ParsingException {
+    Document document = parseOutputFile("dependency_trees.html");
+    Nodes dependencyTreeParagraph = document.query("//p[@class='dependency-tree-node']");
+
+    Assert.assertEquals(53144, dependencyTreeParagraph.size());
```

Comment:
This test is set to use libraries-bom:1.0.0. Therefore this number 53144 does not change when we make changes to libraries-bom.

Reply:
However it appears it does change when we drop transitive optional dependencies. Not unexpected.

Reply:
Yes, that's correct.